includes/admin/give-metabox-functions.php

```diff
@@ -796,29 +796,6 @@ function give_get_field_description( $field ) {
 
 
 /**
- * Get field custom attributes as string.
- *
- * @since 1.8
- *
- * @param $field
- *
- * @return string
- */
-function give_get_custom_attributes( $field ) {
```

Comment:
@DevinWalker This function deprecated. I can use `give_get_attribute_str` fx instead